lib/profile/commands/view.rb

```diff
@@ -11,15 +11,43 @@ class View < Command
       def run
         @name = args[0]
         raise "Node '#{@name}' not found" unless node
+
+        if @options.watch
+          in_clean_window do
+            loop do
+              height = `tput lines`.chomp.to_i
+              print "\r\e[#{height}A"
```

Comment:
What is this line doing? I'm not familiar with the `\e` escape sequence.

Reply:
`\e[<number>A` will move the terminal cursor `<number>` lines upwards (if my understanding is correct). It's being used in combination with `\r` to move the terminal cursor to the top-left of the window so that the next `puts` will print from the top of the screen.

Reply:
Ah, I missed the `[`. That is indeed what it seems to be doing :smile: